### DISCUSSION TITLE
Feature: Login Banner Request

### DISCUSSION THREAD

**Discussion by darth-drew** [Security-Onion-Solutions/securityonion#3680] [Unsupported Versions] [answered]
Bringing up #2643 again. The way I read #2647 it was implemented in 2.3.30, but I haven't been able to find further information as of yet. For my purposes, a text field located under the red mark would be great, but a checkbox showing users have agreed to the terms would be extra nice.
![image](https://user-images.githubusercontent.com/12967094/113012929-3acbb180-9149-11eb-9149-1c3e4231188b.png)

**Comment by jertel** [comment, marked as answer]
An undocumented feature was made available in 2.3.30 but it's not something we recommended using due to upgrade incompatibilities. However, in the next release, currently slated as 2.3.50, you will be able to customize a banner.md Markdown file. The contents of that banner file will be displayed above the Email Address input field on the login form. You could add a checkbox via HTML inside of the banner.md file if you wanted, but that value would not have any functional connection to storage so it would be purely cosmetic.

**Reply by darth-drew** [reply]
Thanks for the reply! Tracking as #3659
